packages/salesforcedx-vscode-lwc/resources/static/js-meta.xsd

```diff
@@ -0,0 +1,291 @@
+                                Enables a component to be used on a record page in Lightning App Builder.
+                            </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="lightning__UtilityBar">
+              <xs:annotation>
+                <xs:documentation>
+                                Enables a component to be used as a utility item on the utility bar in the App Manager.
+                            </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="lightning__FlowScreen">
+              <xs:annotation>
+                <xs:documentation>
+                                Enables a component to be used on flow screens in Flow Builder.
+                            </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="lightning__Tab">
+              <xs:annotation>
+                <xs:documentation>
+                                Enables a component to be used in a custom tab in Lightning Experience or the Salesforce mobile app.
+                            </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="lightning__Inbox">
+              <xs:annotation>
+                <xs:documentation>
+                                Enables a component to be used in Lightning App Builder to add to email application panes for the Outlook and Gmail integrations.
+                            </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="lightningCommunity__Page">
+              <xs:annotation>
+                <xs:documentation>
+                                Enables a component to be used on a Lightning community page in Experience Builder.
+                            </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="lightningCommunity__Default">
+              <xs:annotation>
+                <xs:documentation>
+                                Used together with lightningCommunity__Page. Enables a component that includes configurable properties to be used on a Lightning community page in Experience Builder. When the component is selected on the page, the properties appear.
+                            </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="lightningSnapin__ChatMessage">
+              <xs:annotation>
+                <xs:documentation>
+                                Enables a component installed from a managed package to run in a community that has Lightning Locker disabled. This setting allows managed package developers to opt in to running their Lightning web components in a community with Lightning Locker disabled.
```

Comment:
Looks like a copy/paste error. Replace with:
Enables a custom chat message component to be selected from Embedded Service Chat Setup. A component that imports the lightningsnapin/baseChatMessage module must specify the lightningSnapin__ChatMessage target. For more information, see the lightningsnapin/baseChatMessage documentation.